### PR TITLE
Add audit columns to endpoint registry DB tables

### PR DESCRIPTION
**Pull request by ChamodDamitha**

### CODE REVIEW

File: components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/EndpointRegistryConstants.java [hunk 1]
```diff
@@ -59,4 +57,12 @@
 
     public static final String COLUMN_METADATA = "METADATA";
 
+    public static final String COLUMN_CREATED_BY = "CREATED_BY";
+
+    public static final String COLUMN_UPDATED_BY = "UPDATED_BY";
```
Comment (fazlan-nazeem) [inline components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/EndpointRegistryConstants.java:62]: in AM_DB tables we have been using UPDATED_TIME , CREATED_TIME. shall we use the same?

Comment (ChamodDamitha) [reply components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/EndpointRegistryConstants.java:62]: Fixed in https://github.com/wso2/carbon-apimgt/pull/8550/commits/d29130c659d23aef2a47475d566e30fa30bcdc56

File: features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/sql/h2.sql [hunk 2]
```diff
@@ -1849,6 +1852,10 @@ CREATE TABLE IF NOT EXISTS ENDPOINT_REG_ENTRY (
   SERVICE_CATEGORY VARCHAR(50),
   ENDPOINT_DEFINITION BLOB,
   REG_ID INTEGER NOT NULL,
+  CREATED_TIME TIMESTAMP,
```
Comment (ruks) [inline features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/sql/h2.sql:1855]: Isn't CREATED_TIME and UPDATED_TIME set  CURRENT_TIMESTAMP in the script? Or is it handled in the java code?

Comment (ChamodDamitha) [reply features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/sql/h2.sql:1855]: Handled in the java code

**Pull request merged**